fix(function): ST_Equals should return true for same type empty geometries (#27015)
prestodb/presto: presto-docs/src/main/sphinx/functions/geospatial.rst

.. function:: ST_Equals(Geometry, Geometry) -> boolean

    Returns ``true`` if the given geometries represent the same geometry
    according to ISO SQL/MM semantics. Also returns ``true`` if both geometries are empty,
    regardless of their geometry types.

>     Note: A legacy implementation is available via the ``legacy_st_equals`` session property

Reviewer: ```suggestion
    Note: A legacy implementation is available by using the ``legacy_st_equals`` session property
```
Avoid Latin abbreviations. See the [GitLab documentation style guide recommended word list entry for via](https://docs.gitlab.com/development/documentation/styleguide/word_list/#via) for discussion and alternatives.

    or ``legacy-st-equals`` configuration property. The legacy behavior returns ``false`` for

>     any two empty geometry comparisons, but otherwise behaves the same. This legacy behavior

Reviewer: ```suggestion
    any two empty geometry comparisons, but otherwise behaves the same. 
```

>     will be removed in a future release.

Reviewer: ```suggestion
```
